src/components/NavBar/ChainSelectorRow.tsx

```diff
@@ -0,0 +1,90 @@
+import { useWeb3React } from '@web3-react/core'
+import Loader from 'components/Loader'
+import { getChainInfo } from 'constants/chainInfo'
+import { SupportedChainId } from 'constants/chains'
+import { CheckMarkIcon } from 'nft/components/icons'
+import styled, { useTheme } from 'styled-components/macro'
+
+const LOGO_SIZE = 20
+
+const Container = styled.button`
+  display: grid;
+  background: none;
+  grid-template-columns: min-content 1fr min-content;
+  align-items: center;
+  text-align: left;
+  line-height: 24px;
+  border: none;
+  justify-content: space-between;
+  padding: 10px 8px;
+  margin: 0 8px;
+  cursor: pointer;
+  border-radius: 12px;
+  color: ${({ theme }) => theme.textPrimary};
+  width: 240px;
```

Comment:
Width is off when left aligned
![Screen Shot 2022-09-20 at 11 30 08 ](https://user-images.githubusercontent.com/11512321/191336595-9c8bf92d-a3c5-46e9-9839-6d14b87a5af6.png)

Reply:
Might actually be right padding
![Screen Shot 2022-09-20 at 11 31 28 ](https://user-images.githubusercontent.com/11512321/191336705-dc9adc67-8151-47ac-aef0-a8121dd338c5.png)